Sources/StreamChatUI/Utils/TextViewMentionedUsersHandler.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class TextViewMentionedUsersHandler {
```

Comment:
The class is open, we need docs for it and the method.

Reply:
Added docs already ✅